node/sc/bridge_manager_test.go

```diff
@@ -92,9 +92,9 @@ func TestBridgeManager(t *testing.T) {
 	wg.Add(6)
 
 	// Config Bridge Account Manager
-	config := &SCConfig{}
+	config := &SCConfig{DefaultGasLimit: 5000000}
```

Comment:
There are too many hard-coded values. Can we set this value using the value from the parameters?

https://github.com/klaytn/klaytn/pull/1287/files#diff-7aa83061ca1c69a8ab9203da49a7581bad6e7cc80ecb96582e7fb09c0b8f6534R745

Reply:
Thanks for the advice. I removed hardcoded value.